setup.py

```diff
@@ -17,19 +17,18 @@
         'License :: OSI Approved :: GNU Affero General Public License v3',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.5',
         'Framework :: Django',
-        'Framework :: Django :: 1.8',
-        'Framework :: Django :: 1.9',
-        'Framework :: Django :: 1.10',
-        'Framework :: Django :: 1.11',
+        'Framework :: Django :: 2.2.12',
+        'Framework :: Django :: 3.0',
+        'Framework :: Django :: 3.1',
     ],
     packages=[
         'edxlearndot',
         'tests'
     ],
     install_requires=[
-        "django >= 1.8, < 2.0",
+        "django",
```

Comment:
This change is going to cause issues when this package is installed in the edxapp venv, because pip will [install the latest version of django](https://pip.pypa.io/en/stable/user_guide/#installing-packages) and override the edx-platform version.

Could you instead pin it to allow the [juniper django version](https://github.com/edx/edx-platform/blob/open-release/juniper.1/requirements/edx/django.txt#L1) to remain installed?

```suggestion
        "django>=2.2, >3",
```

Reply:
Thanks for bringing this into my attention, I didn't know about that, but now I do :smile:

Reply:
I'm sorry for not committing the suggestions that you did, I just wanted to group the commits together, so that I don't do multiple tweaks in different commits for the same cause (or force push).

Reply:
No worries @nizarmah -- I'm not fussed if you accept the suggestions as-is or just do them yourself :)

Reply:
Yeah, but you put time and effort into the review and the suggestions and I value letting the person know that I appreciate their help (: